elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -64,7 +73,7 @@ See mixin blocks before selectors that contain the selector name and the three l
   left: 0;
   width: 100%;
   height: 100%;
-  z-index: -1;
+  z-index: 97;
```

Comment:
Instead of setting this separately, if you add a z-index to the `pfe-navigation` tag and leave this at -1 it'll produce the same effect without making two z-indexes we have to manage.

Parent elements that are 'positioned' and have a z-index will 'scope' children elements. So this will be -1 in pfe-nav, not -1 in the page.

Reply:
@wesruv Oh cool, thanks for explaining that. I will get that updated.

Reply:
@wesruv added those changes.